xgi/utils/tensor.py

```diff
@@ -0,0 +1,191 @@
+## Tensor times same vector in all but one (TTSV1) and all but two (TTSV2)
+from collections import defaultdict
+from itertools import combinations
+from math import factorial
+
+import numpy as np
+from numpy import prod
+from scipy.signal import convolve
+from scipy.sparse import coo_array
+from scipy.special import binom as binomial
+
+__all__ = [
+    "pairwise_incidence",
+    "ttsv1",
+    "ttsv2",
+]
+
+
+def pairwise_incidence(edgedict):
+    """Create pairwise adjacency dictionary from hyperedge list dictionary
```

Comment:
Function name says "incidence" but description says "adjacency", maybe pick one. 

Also, this feel like it could/should be a general function? If I got it right, we have `xgi.adjacency_matrix(H, order=1)` that associates a pair of nodes to the number of edges of which it's part, and here we associated it to the list of edge IDs, with a dict. Would it make sense to have a new one like `xgi.adjacency_dict(H, order)` somewhere (not in utils/tensor.py)?

Reply:
You're right, we should stick with one convention, and it should be incidence. Yeah, it might make sense to have to have an xgi.incidence_dict(H, order) function.

Reply:
After looking more into this, I think that it's best to stay in the `tensor` utilities module. Updating the docstring right now.